tests/IntegrityTests/ValidatePrereleaseTxt.cs

```diff
@@ -101,7 +101,7 @@ private bool HasPrereleasePackages(string projectPath, IEnumerable<string> packa
 
             return packageRefs
                 .Where(pkgRef => packageNames.Contains(pkgRef.Attribute("Include").Value, StringComparer.OrdinalIgnoreCase))
-                .Any(pkgRef => pkgRef.Attribute("Version").Value.Contains("-") || pkgRef.Attribute("Version").Value.StartsWith("0."));
+                .Any(pkgRef => pkgRef.Attribute("Version").Value.Contains("-"));
```

Comment:
Exploration TF-302: Packages released as 0.x w/o a [pre-release suffix](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions) (`-prerelease`), see [section 9](https://semver.org/#spec-item-9)

**would not be considered prerelease.** 

[The rationale](https://semver.org/#spec-item-4) is SemVer spec which does not identify 0.x releases are pre-releases:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

A release 0.x w/o the suffix should not contain the `prerelease.txt` marker file for the documentation.

With the Preview Program in place, the expectations is that there will be no 0.x releases that are not part of the Preview Program.